tests/unit/test_tf_estimator.py

```diff
@@ -174,6 +178,7 @@ def test_tf(time, strftime, sagemaker_session):
     assert {'Environment':
             {'SAGEMAKER_SUBMIT_DIRECTORY': 's3://{}/{}/sourcedir.tar.gz'.format(BUCKET_NAME, JOB_NAME),
              'SAGEMAKER_PROGRAM': 'dummy_script.py',
+             'SAGEMAKER_REQUIREMENTS': 'requirements.txt',
```

Comment:
What happens with the requirements file is in a different folder than the user_script?
What happens if the user passes relative or absolute paths?
What happens if the user passes an invalid path?

We need unit test cases for that.

Reply:
we can add validation (with corresponding unit tests) for that. I think it'd be sufficient to check that the path is not relative and that the file exists using `os.path.exists()`

Reply:
Yeah, I think that will be sufficient. Thanks.